src/util/std_types.h

```diff
@@ -1082,6 +1089,13 @@ inline bool can_cast_type<bitvector_typet>(const typet &type)
          type.id() == ID_c_bool;
 }
 
+/// This method tests,
+/// if the given typet is a signed or unsigned bitvector.
+inline bool is_signed_or_unsigned_bitvector(const typet &type)
+{
+  return type.id() == ID_signedbv || type.id() == ID_unsignedbv;
+}
```

Comment:
By now I'd suggest to make this consistent with `integer_bitvector_type`, doing something like `is_integer_bitvector_type`.